### PR TITLE
Admin autosuggest

### PR DESCRIPTION
**Pull request by jvmsangkal**

### CODE REVIEW

File: app/scripts/stores/admin/music_provider_store.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ var Reflux = require('reflux'),
         loadMusicProvider: function () {
             var self = this;
 
-            music_api.music_provider.all(function (err, result) {
+            music_api.music_provider.all('','','',function (err, result) {
```
Comment (sibayanjasper) [inline app/scripts/stores/admin/music_provider_store.js]: What's this?

File: app/scripts/stores/autocomplete/user_store.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+'use strict';
+var Reflux = require('reflux'),
+    music_api = require('api/music_api'),
+
+    UserStore = Reflux.createStore({
+        listenables: [],
```
Comment (sibayanjasper) [inline app/scripts/stores/autocomplete/user_store.js]: Is this needed?

File: app/scripts/components/admin/music_provider/modals/add_mp_modal.js [hunk 3]
```diff
@@ -40,12 +41,13 @@ var React = require('react'),
...
-                            element_id="email"
-                            element_value=""
-                            element_classes="validate"
-                            element_label="Owner Email Address"
-                            required="true" />
+                        <AutosuggestText ref="email"
+                            id="email"
+                            required={true}
+                            label="Owner Email Address"
+                            placeholder=""
```
Comment (sibayanjasper) [inline app/scripts/components/admin/music_provider/modals/add_mp_modal.js]: Why no placeholder?

File: app/scripts/stores/admin/music_provider_store.js (hunk 1 above)
Comment (jvmsangkal) [inline app/scripts/stores/admin/music_provider_store.js]: these are the parameters in music api (page, entries, count_pages, callback). nagiging page si callback kay nilagay ko.:)

File: app/scripts/components/admin/music_provider/modals/add_mp_modal.js [hunk 4]
```diff
@@ -40,12 +41,13 @@ var React = require('react'),
...
+                            value=""
```
Comment (sibayanjasper) [inline app/scripts/components/admin/music_provider/modals/add_mp_modal.js]: Remove attributes if no value

File: app/scripts/stores/admin/music_provider_store.js (hunk 1 above)
Comment (sibayanjasper) [inline app/scripts/stores/admin/music_provider_store.js]: Why set to empty strings?

File: app/scripts/components/admin/music_provider/modals/add_mp_modal.js (hunk 3 above)
Comment (sibayanjasper) [inline app/scripts/components/admin/music_provider/modals/add_mp_modal.js]: Remove the placeholder

**Pull request merged**